texar/modules/decoders/decoder_base.py

```diff
@@ -65,6 +70,7 @@ def _make_output_layer(layer: Optional[Union[nn.Module, torch.Tensor]],
                 "wanted.")
         output_layer = nn.Linear(output_size, vocab_size, bias)
     elif torch.is_tensor(layer):
+        assert isinstance(layer, torch.Tensor)
```

Comment:
`torch.is_tensor` is actually just `isinstance(_, torch.Tensor)`. We can just replace this function call with `isinstance`.